src/model/channel/channel_id.rs

```diff
@@ -710,6 +710,7 @@ impl ChannelId {
     /// [`CreateMessage::content`]: crate::builder::CreateMessage::content
     /// [Attach Files]: Permissions::ATTACH_FILES
     /// [Send Messages]: Permissions::SEND_MESSAGES
+    /// [`File`]: std::fs::File
```

Comment:
Tokio's `File` is used for `Into<AttachmentType>`.  See example.
```suggestion
    /// [`File`]: tokio::fs::File
```